src/decoder/Control.hxx

```diff
@@ -121,6 +121,11 @@ private:
 	 */
 	const AudioFormat configured_audio_format;
 
+	/**
+	 * The "selective_44k_resample" setting.
```

Comment:
This is boilerplate text without any content.

Reply:
Just copying style from the private member above it. I that the "setting" context here meant that is really coming from a value from the setting file (mpd.conf).